src/Cards/Cards.cpp

```diff
@@ -1,5 +1,141 @@
-//
-// Created by samue on 2024-09-12.
-//
-
 #include "Cards.h"
+#include <cstdlib>  // For rand() and srand()
+#include <ctime>  // For time()
+
+// Card Class Implementation
```

Comment:
Also should put your work in a common namespace for good practice and none of our naming will overlap.